java/pom.xml

```diff
@@ -158,6 +159,28 @@
                 <cxx.flags>-Wno-deprecated-declarations</cxx.flags>
             </properties>
         </profile>
+        <profile>
+            <id>no-cufile-tests</id>
+            <activation>
+                <property>
+                    <name>USE_GDS</name>
```

Comment:
Not a criticism just a note. We need to make sure that the release build scripts are working after this change. In maven if you manually enable a profile on the command line with `-P` it turns off all other profiles (I believe even if they would have been activate automatically) and you have to specify all of them.  This just means that if we try to do a release build with both `-Prelease` and  `-DUSE_GDS=ON` the second one might be ignored.